recipes-bsp/firmware-pack/firmware-pack.bb

```diff
@@ -0,0 +1,52 @@
+SECTION = "bootloaders"
+SUMMARY = "Firmware Packaging"
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/BSD-3-Clause;md5=550794465ba0ec5312d6919e203a55f9"
+
+inherit deploy
+
+PACKAGE_ARCH = "${MACHINE_ARCH}"
+
+DEPENDS = " trusted-firmware-a u-boot bootparameter-native fiptool-native"
+
+S = "${WORKDIR}/sources"
+UNPACKDIR = "${S}"
```

Comment:
You don't seem to use the UNPACKDIR variable. Is it redundant?

Reply:
Updated in the commit: 5a26648ad6f24593e5f2cf10767ae3231b2b07b7